tasks/auditd.yml

```diff
@@ -4,6 +4,7 @@
   package:
     name: '{{ auditd_package }}'
     state: 'present'
+  tags: auditd
```

Comment:
These are not necessary as there's already a tag here: https://github.com/dev-sec/ansible-os-hardening/blob/master/tasks/hardening.yml#L17

Reply:
alright thanks for the hint. did not see that. I removed the redundant tag.